.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Delete this  `.vscode/settings.json`

Reply:
@iamsunil25

Reply:
okay

Reply:
@Aryamanz29 i deleted setting.json file as you said